TMDbLib/Objects/TvShows/NetworkBase.cs

```diff
@@ -9,5 +9,11 @@ public class NetworkBase
 
         [JsonProperty("name")]
         public string Name { get; set; }
+
+        [JsonProperty("logo_path")]
+        public string LogoPath { get; set; }
```

Comment:
Network (inherits NetworkBase) does not describe a Logo_path in its [API docs](https://developers.themoviedb.org/3/networks/get-network-details).

Reply:
Should be fixed 😄